core/src/cubos/core/io/glfw_window.hpp

```diff
@@ -37,9 +37,15 @@ namespace cubos::core::io
         void setCursor(std::shared_ptr<Cursor> cursor) override;
         void setClipboard(const std::string& text) override;
         const char* getClipboard() const override;
+        bool keyPressed(Key key) const override;
+        bool keyPressed(Key key, Modifiers modifiers) const override;
+
+        Modifiers modifiers() const;
+        void modifiers(Modifiers modifiers);
```

Comment:
Why would this be used externally?

Reply:
It isn't, but I need it in static functions that receive the window as an argument